PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+I want to merge this change because...
+
+(Please mention all relevant issue numbers.)
+
+(If your changes affect the UI, providing "before" and "after" screenshots will
+greatly reduce the amount of work needed to review your work.)
+
+### Pull Request Checklist
+
+(Please keep this section. It will make maintainer's life easier.)
+
+1. [ ] Privileged views and APIs are guarded by proper permission checks.
+1. [ ] All visible strings are translated with proper context.
```

Comment:
If we add too much here developers will give up too easily. Especially we don't have proper context in our code and we are require that from others.

Reply:
Maybe it's not clear but we don't require that you personally check all of those checkboxes. They are criteria for merging.